migration-engine/cli/tests/cli_tests.rs

```diff
@@ -115,6 +145,26 @@ fn test_create_database(api: TestApi) {
     assert!(output.status.success());
 }
 
```

Comment:
if drop now works this test might be unnecessary. There is one that tests create for Mysql and Pg already, this was only special cased since drop was not implemented yet. It's probably enough to enable MSSQL and the common create test now.